src/sql/procedure/insert_raw_employee_data.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO raw_employee_data
+VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s);
```

Comment:
Same here. The insert query is never going to change, the only thing varying here is the values and the way you map field.
You can look into mapping JSON, CSV and XML files, because it is not guaranteed that you will received data always in the same patten.